payara/Payara
# FISH-8208 : year in index.html updated

## Description
Update copyright year in Payara's default index.html

## Important Info
### Blockers
none

## Testing
### New tests
none

### Testing Performed
1. Build Payara Server
2. Run Payara Server
3. Poke http://localhost:8080/
4. Copyright 2024 should appear at the bottom of the page

### Testing Environment
Zulu JDK 11 on Windows 11 with Maven 3.8.6

## Documentation
None
